script.js

```diff
@@ -0,0 +1,27 @@
+function check(number) {
+    if(number=== "" || isNaN(number)) {
```

Comment:
```suggestion
    if(number=== "" || typeof number !== "number") {
```

`typeof` is more common. `isNaN` also do a job but is very rare